slotify/authentication/util.py

```diff
@@ -0,0 +1,29 @@
+from django.urls import reverse
+from django.core.mail import EmailMessage
+from rest_framework_simplejwt.tokens import RefreshToken
+from .models import User
+
+class RegistrationUtil:
+    @staticmethod
+    def send_email_verification(user: User, domain: str):    
+        access_token = RefreshToken.for_user(user).access_token    
+        relative_link = reverse('verify-email')
+
+        # TODO: replace with https eventually
+        verification_link = f"http://{domain}{relative_link}?token={str(access_token)}"
```

Comment:
Is the http issue only because of localhost? I guess https should be ok in production as our server is configured to use https. If that's the case perhaps we can use a environment variable to deal with the differences.